sdk/identity/azure-identity/tests/recording_processors.py

```diff
@@ -40,12 +43,19 @@ def process_response(self, response):
         except (KeyError, ValueError):
             return response
 
-        for field in body:
-            if field in SECRETS:
-                # record a hash of the secret instead of a simple replacement like "redacted"
-                # because some tests (e.g. for CAE) require unique, consistent values
-                digest = hashlib.sha256(six.ensure_binary(body[field])).digest()
-                body[field] = six.ensure_str(binascii.hexlify(digest))
+        with open(MGMT_SETTINGS_FAKE_IDENTITY, "w+") as outfile:
```

Comment:
What should we generate this each time? What happens if this file ends up getting checked in?